core-services/release-controller/_releases/release-ocp-4.10-ci.json

```diff
@@ -47,7 +47,6 @@
     },
     "upgrade":{
       "upgrade":true,
-      "optional":true,
```

Comment:
this is covered better by the aggregated job below

Reply:
so that's "please leave this `optional`, because the already-required `aggregated-gcp-sdn-upgrade-4.10-micro` has patch bumps sufficiently covered?  Do we need to run an aggregated job if the one-shot job is sufficiently successful on its own?

Reply:
also, checking [a recent aggregator run][1]:

```
: [sig-arch][Feature:ClusterUpgrade] Cluster should remain functional during upgrade [Disruptive] [Serial] | 0s
-- | --
name: '[sig-arch][Feature:ClusterUpgrade] Cluster should remain functional during   upgrade [Disruptive] [Serial]' summary: 'Passed 0 times, failed 10 times, skipped 0 times: we require at least one   pass to consider it a success' passes: [] failures: - jobrunid: "1441385880848699392"   humanurl: https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.10-e2e-gcp-upgrade/1441385880848699392   gcsartifacturl: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.10-e2e-gcp-upgrade/1441385880848699392/artifacts - jobrunid: "1441385880894836736"
...
```

But digging into [that job][2], it is a success, despite the suite not bothering to mark that failure as a flake:

![image](https://user-images.githubusercontent.com/209920/134728743-0fd3eb3f-94a1-46bf-ad08-be5c10f87173.png)

Or maybe the single-shot jobs are not broken, and that should not have been `passed`?

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/aggregated-gcp-sdn-upgrade-4.10-micro-release-openshift-release-analysis-aggregator/1441385882480283648
[2]: https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.10-e2e-gcp-upgrade/1441385880848699392

Reply:
Single-shot jobs were falsely happy.  #22331 will fix that.

Reply:
> Do we need to run an aggregated job if the one-shot job is sufficiently successful on its own?

Yes, for antiflakes.  We're looking to be "X% reliable", not "can pass at least once"

Reply:
This isn't going to be "can pass at least once", is it?  This is going to be "fail the nightly if it fails even once".